ui/tests/acceptance/auth/test-helper.js

```diff
@@ -3,13 +3,10 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { click, currentURL, fillIn } from '@ember/test-helpers';
+import { click, currentURL } from '@ember/test-helpers';
 import { GENERAL } from 'vault/tests/helpers/general-selectors';
-
-const SELECTORS = {
-  mountType: (name) => `[data-test-mount-type="${name}"]`,
-  submit: '[data-test-mount-submit]',
-};
+import { MOUNT_BACKEND_FORM } from 'vault/tests/helpers/components/mount-backend-form-selectors';
+import { mount } from 'vault/tests/helpers/components/mount-backend-form-helpers';
```

Comment:
naming here. i kept it what it was before in the page object. While it's accurate it's not super descriptive. It mounts both secret engines and auth methods. Any strong feelings about renaming, and if so, any suggestions?